frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -780,6 +843,14 @@ function ReaderRolling:onGotoPrevChapter()
     return true
 end
 
+function ReaderRolling:onContentSelection()
+    return self.ui.highlight:onStartHighlightIndicator()
+end
+
+function ReaderRolling:onToggleWifi()
+    return self.ui.networklistener:onToggleWifi()
+end
+
```

Comment:
Dunno if all these (the last 3 added here and similar everywhere else) could get a more explicite name that they are key_events targets (ie. `onKeySomething`) - so we don't get to have them used by Dispatcher/Gestures and our more global events.